### DISCUSSION TITLE
Welcome to Codeathon 2023!

### DISCUSSION THREAD

**Discussion by anjohnson** [epics-base/epics-base#315] [Codeathon 2023]
Welcome to the EPICS Codeathon 2023, which will be held in-person at Diamond Light Source from 6-10 March but will also accept virtual attendees.

This discussion category can be used for any topics related to this year's Codeathon.

See the [Codeathon Wiki](https://github.com/epics-base/epics-base/wiki/Codeathon-project-ideas) page for some project ideas. Other modules such as [Asyn](https://github.com/epics-modules/asyn), [IOC Status](https://github.com/epics-modules/iocStats), [autosave](https://github.com/epics-modules/autosave) and [caPutLog](https://github.com/epics-modules/caPutLog) may also have issues that could be worked on.

**Comment by anjohnson** [comment]
Transport from Heathrow to Wantage may be most efficient by taking a [the airline OXF](https://www.oxfordbus.co.uk/services/OXBC/OXF?date=2023-03-05&direction=inbound) coach from Terminal 5 or the Central Bus Terminal to Oxford's Gloucester Green Bus station, then a [Stagecoach S9](https://www.oxfordbus.co.uk/services/SCOX/S9?date=2023-03-05&direction=outbound) bus to Wantage Market Place. Frequency of busses might be an issue later in the day, check the above links for times, or take a Taxi from Oxford to Wantage. Personally I find Google Maps fairly useful for public transport details.

Going by train may require several changes, including travel into central London (the Heathrow Express costs more but goes straight to Paddington, the Underground is slower and cheaper but you have to change lines to get to Paddington) then back out to Didcot Parkway. The [connector X32 bus](https://www.oxfordbus.co.uk/services/THTR/X32?date=2023-03-04&direction=outbound) goes to Wantage (via Diamond) from Didcot Parkway, or take a Taxi.

**Reply by FreddieAkeroyd** [reply]
You can also take the Railair coach to Reading and then catch a train to Didcot followed by either bus or taxi to Wantage. https://www.railair.com/timetables  

If you like trains an alternative to in and out of london is to catch the elizabeth line https://content.tfl.gov.uk/elizabeth-line-map.pdf to Hayes and Harlington and then change platform and catch a train travelling the other direction towards reading. Some london to didcot trains stop at Hayes, or catch a train to Reading and then another onwards to Didcot. Only comments i can make about Hayes is that it didn't used to have a platform lift, so was not a good change if you have a heavy suitcase. However a google suggests it does have a lifts now, but i can't personally confirm that.

A good site for travel connections is https://www.traveline.info/

**Comment by anjohnson** [comment]
## Video Conference Sessions

A Zoom video conference room "V/0-0 - FHI Zoom Room 01" will be available for collaboration with remote participants, kindly provided by Heinz Junkes and the Fritz Haber Institute (FHI).

This should be open daily from Monday through Friday between 06:00 and 22:00 (those times might be in the Central European time zone though, not GMT).

The direct link to the Zoom meeting has been removed.
